pymc3/plots.py

```diff
@@ -168,34 +167,53 @@ def autocorrplot(trace, vars=None, max_lag=100, burn=0, ax=None):
         
     Returns
     -------
-
     ax : matplotlib axes
-
     """
     
     import matplotlib.pyplot as plt
-
+        
+    def _handle_array_varnames(val):
+        #val = str(val)
```

Comment:
This is what I'm saying, the commented out line 176 is what I put in to deal with the original, single failed test. That addition led to almost all tests failing with the theano errors I cannot figure out,  and now when I comment the line back out I still get all the theano errors. So I've "magically" gone from a small problem to a seemingly large one with no change.

The current test (that I'm now failing yet again) has all the labeling commented out, which was the only other alteration I made with, but I cannot see how something as benign as ```ax.set_title()``` should cause any test to fail.

I'll persist, but I really would like some insight into these theano errors if anyone has any ideas :-)